src/main/java/org/octri/hpoonfhir/service/Stu2FhirService.java

```diff
@@ -31,13 +33,35 @@ public FhirContext getFhirContext() {
 		return ctx;
 	}
 	
+	@Override
+	public Patient findPatientById(String id) throws FHIRException {
+		org.hl7.fhir.instance.model.Patient stu2Patient = getClient().read().resource(org.hl7.fhir.instance.model.Patient.class).withId(id).execute();
```

Comment:
Sorta weird you had to include the full classpath here.

Reply:
The point of this service is to convert from STU2 responses to STU3 objects. They all have the same names with different packages, and I am using both in the class.

Reply:
Further reading if you're interested: http://hapifhir.io/doc_converter.html